### ISSUE TITLE
Timezone incorrect in boot2docker VM

### ISSUE THREAD

**Issue by y3ddet** [Parallels/docker-machine#3]
By default, the VM is configured to sychronize time to the host, however the timezones are mismatched. The VM is set for UTC, while the host OS X is set for a local timezone (MDT here). As a result, when the VM is first started it sets both clocks to the same wall number (12:30pm MDT and 12:30pm UTC) which causes a mismatch.

The first indication of this failure is that the TLS certificate created by docker-machine for access to the Docker daemon will fail. Opening the certificate file with openssl shows the "not-valid before" time in UTC, and with the host on a different timezone, certificate checking fails until the VM time is corrected.

**Comment by legal90** [comment]
Yes, seems like we have a bug in "Time Sync" feature. 
I'll temporary disable it by default, but it means that we have to release a new custom binary of `docker-machine`. I will do it in 1-2 days. 

You can fix it manually for your VM in the moment:
```console
docker-machine stop <vm_name>
prlctl set prl-test --time-sync off
docker-machine start <vm_name>
```

P.s. Boot2Docker guest OS has an internal way to sync the time (it runs ntp by cron job). So its time will be always correct even if "Time Sync" feature of Parallels Desktop will be disable.

**Comment by y3ddet** [comment]
Great, at least this workaround will allow me to run Make on my projects again (make does not like clock problems...)

**Comment by y3ddet** [comment]
Actually, not good. The internal NTD process is also fracked:

The following commands were run within the VM about every 8 seconds. 

> Fri Aug 21 23:32:47 UTC 2015
> minion@0a87fa556f55:/build$ date
> Fri Aug 21 23:32:54 UTC 2015
> minion@0a87fa556f55:/build$ date
> Sat Aug 22 00:34:11 UTC 2015

It seems that whenever the NTP cron is run, the time is advanced by 1 hour... at least it is not going backwards which is what happens if you try to manually correct the time by running ntpclient.

**Comment by legal90** [comment]
Fixed with ad586e86 and released as [0.4.0-1](https://github.com/Parallels/docker-machine/releases/tag/parallels%2F0.4.0-1)

```console
$ curl -L https://github.com/Parallels/docker-machine/releases/download/parallels%2F0.4.0-1/docker-machine_darwin-amd64 > /usr/local/bin/docker-machine
$ chmod +x /usr/local/bin/docker-machine
```

@y3ddet Please, check that it works for you (you will need to re-create the machine from the scratch)

**Comment by y3ddet** [comment]
@legal90 Confirmed! Clock is now correctly showing actual UTC in the container, and the clock is stable over the last 4 hours.

The older container with the bug kept spinning forward, gaining about 48-hours for every real 24-hours, resulting in a clock that was more than 7+ days faster than wall clock after the weekend!
